File: usage-case/api/src/test/java/org/luvx/coding/jdk/concurrent/MainTest.java
package org.luvx.concurrent;

import org.junit.jupiter.api.Test;

import java.util.Arrays;
import java.util.concurrent.ThreadLocalRandom;
import java.util.stream.IntStream;

public class MainTest {
    @Test
    void m1() {
        int[] array = IntStream.rangeClosed(0, 100)
                .map(i -> ThreadLocalRandom.current().nextInt(10))
                .toArray();
        System.out.println(Arrays.toString(array));
    }
}

File: usage-case/api/src/test/kotlin/org/luvx/coding/guava/MainTest.kt
package org.luvx.coding.guava

import com.google.common.collect.Range
import com.google.common.util.concurrent.RateLimiter
import org.junit.jupiter.api.Test
import org.luvx.coding.common.enhancer.out
import java.time.LocalDateTime

internal class MainTest {

    @Test
    fun m2() {
        val create = RateLimiter.create(1.0)
        LocalDateTime.now().out()
        create.acquire()
        LocalDateTime.now().out()


    }

    @Test
    fun m3() {
        val range = Range.closed(1, 20)
        range.out()
    }
}